ReactAndroid/src/main/java/com/facebook/react/uimanager/JSTouchDispatcher.java

```diff
@@ -149,6 +157,7 @@ public void handleTouchEvent(MotionEvent ev, EventDispatcher eventDispatcher) {
         );
       }
       mTargetTag = -1;
+      mGestureStartTime = Long.MIN_VALUE;
```

Comment:
Small thing: instead of using Long.MIN_VALUE directly, it'd be nice to specify a final static long named `UNSET` (or something like that) that we then use and check against.